packages/core-data/src/utils/crdt-blocks.ts

```diff
@@ -184,13 +179,13 @@ function createNewYBlock( block: Block ): YBlock {
  * This function is called to sync local block changes to a shared Y.Doc.
  *
  * @param yblocks        The blocks in the local Y.Doc.
- * @param incomingBlocks Gutenberg blocks being synced, either from a peer or from the local editor.
- * @param lastSelection  The last cursor position, used for hinting the diff algorithm.
+ * @param incomingBlocks Gutenberg blocks being synced.
+ * @param cursorPosition The position of the cursor after the change occurs.
  */
 export function mergeCrdtBlocks(
-	yblocks: YBlocks,
-	incomingBlocks: Block[],
-	lastSelection: WPBlockSelection | null
+	yblocks: Y.Array< YBlock >, // yblocks represent the blocks in the local Y.Doc
```

Comment:
can you revert this type change?

```suggestion
	yblocks: YBlocks, // yblocks represent the blocks in the local Y.Doc
```

Reply:
Fixed (along with the duplicate comments) in https://github.com/WordPress/gutenberg/pull/72604/commits/b89cffba9460fd204842203f0b964848f6d8cd35